specs/fjord/derivation.md

```diff
@@ -48,3 +54,49 @@ be in the same epoch as the the last pre-Fjord blocks, even if these blocks woul
 have these L1-origin timestamps according to pre-Fjord rules. So the same L1 timestamp would be
 shared within a pre- and post-Fjord mixed epoch. This is considered a feature and is not considered
 a security issue.
+
+## Increasing `MAX_RLP_BYTES_PER_CHANNEL` and `MAX_CHANNEL_BANK_SIZE`
+
+With Fjord, `MAX_RLP_BYTES_PER_CHANNEL` will be increased from 10,000,000 bytes to 100,000,000 bytes,
+and `MAX_CHANNEL_BANK_SIZE` will be increased from 100,000,000 bytes to 1,000,000,000 bytes.
+
+The usage of `MAX_RLP_BYTES_PER_CHANNEL` is defined in [Channel Format](../protocol/derivation.md#channel-format).
+The usage of `MAX_CHANNEL_BANK_SIZE` is defined in [Channel Bank Pruning](../protocol/derivation.md#pruning).
+
+The new value will be used when the timestamp of the L1 origin of the derivation pipeline >= the Fjord activation
+timestamp. (TODO: Link to L1 origin activation like in Canyon CB changes).
+
+
+### Rationale
+
+A block with a gas limit of 30 Million gas has a maximum theoretical size of 7.5 Megabytes by being filled up
+with transactions have only zeroes. Currently, a byte with the value `0` consumes 4 gas.
+If the block gas limit is raised above 40 Million gas, it is possible to create a block that is large than
+`MAX_RLP_BYTES_PER_CHANNEL`.
+L2 blocks cannot be split across channels which means that a block that is larger than `MAX_RLP_BYTES_PER_CHANNEL`
+cannot be batch submited.
+By raising this limit to to 100,000,000 bytes, we can batch submit blocks with a gas limit of up to 400 Million Gas.
+In addition, we are able to improve compression ratios by increasing the amount of data that can be inserted into a
+single channel.
+With TODO compression ratio over 6 blobs, we are currently submitting XX MB of compressed data & XX MM of uncompressed
+data per channel.
+This will allow use to use up to XX blobs per channel.
+
+
+Raising `MAX_CHANNEL_BANK_SIZE` is helpful to ensure that we are able to process these larger channels. We retain the
+same ratio of 10 between `MAX_RLP_BYTES_PER_CHANNEL` and `MAX_CHANNEL_BANK_SIZE`.
+
+### Security Considerations
+
+Raising the these limits increases the amount of resources a rollup node would require.
+Specifically nodes may have to allocate large chunks of memory for a channel and will have to potentially allocate more
+memory to the channel bank.
+`MAX_RLP_BYTES_PER_CHANNEL` was originally added to avoid zip bomb attacks.
+The system is still exposed to these attacks, but these limits are straightforward to handle in a node.
+
+These limits also will require more resources in a Fault Proof environemnt when is more constrained than a typical node.
+The change in `MAX_CHANNEL_BANK_SIZE` is not relevant to the first implementation of FP because this limit only tells
+the node when to start pruning & once memory is allocated in the FPVM, it is not garbage collected.
+
+The increase the `MAX_RLP_BYTES_PER_CHANNEL` could cause more resource usage in FPVM; however, we consider this increase
```

Comment:
"of the"?

Is @ajsutton fine with this increase? Does the resource usage raise only to 100MB for the FPVM, or more by some factor?